package.json

```
@@ -31,7 +31,7 @@
     "test:coverage": "cat ./coverage/lcov.info | coveralls"
   },
   "dependencies": {
-    "ms": "2.1.2"
+    "ms": "2.1.3"
```

```suggestion
    "ms": "^2.1.3"
```
to avoid the same problem in the future, using `^` will ensure it gets up to date with new minor/patch versions, which are backwards compatible